.github/actions/load-ci-env/action.yaml

```diff
@@ -0,0 +1,8 @@
+name: Load and common environment variables from ci-config branch
+
+runs:
+  using: composite
+  steps:
+    - name: fetch and set ci config env
+      shell: bash
+      run: curl https://raw.githubusercontent.com/aptos-labs/aptos-core/ci-config/ci.env >> $GITHUB_ENV
```

Comment:
Nit: forge job seems to have failed below because it's using the old wrapper. https://github.com/aptos-labs/aptos-core/pull/3337

Using separate branch to configure this is pretty useful tho

Reply:
fixed

Reply:
Why a branch btw rather than an entirely different github repo for config?

Reply:
I thought about it. It's just a madder of taste/preference. I felt keeping it in same repo for colocation is a slightly nicer, but I'm open to change it to a separate repo if that's what you and others prefer.